.ado/templates/e2e-test-job.yml

```diff
@@ -1,18 +1,33 @@
 #
```

Comment:
The original PR - https://github.com/microsoft/react-native-windows/pull/6686/files - didn't touch this file (or it was renamed since), so this must have from another commit.
Not sure if this diff should be kept here at all

Reply:
@asklar Should I keep this bit or remove it?

Reply:
@dannyvv I see you've made a bunch of changes to the pipeline (perhaps also deleted the `jobs` folder). 
Can you help make sense of what this cherry-pick should look like?

Reply:
The CI didn't even run for this PR, so something is definitely off.

Reply:
Yes, I would remove the file. You can see more if you look at the UI in Azure Devops, but if it does not run, it usually means there was a YAML parse error (such as a file path to a template that does not exist).